src/errors.js

```diff
@@ -11,6 +11,10 @@
 
 import type { BabelNodeSourceLocation } from "babel-types";
 
+// Information: Just an informative message with no semantic implications whatsoever
+// Warning: We came across something that has a well-defined meaning, but is likely to be wrong anyway.
```

Comment:
Prepack will produce code that matches the behavior of the original code, but the original code might have an error.